upgrade/v121to122/upgrade.go

```diff
@@ -0,0 +1,140 @@
+package v121to122
```

Comment:
We should add this to v122to123 instead, because 1.2.2 is already released.

Reply:
Okay!